MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -468,8 +468,8 @@ QStringList MantidUI::getAlgorithmNames() {
   QStringList sl;
```

Comment:
Can this be reserved after `sv` is retrieved to avoid additional memory allocations?

`sl.reserve(sv.size());`